app/pages/event1/member/BenefitsPaidEarlyPage.scala

```diff
@@ -29,4 +31,11 @@ case object BenefitsPaidEarlyPage extends QuestionPage[String] {
 
   override def route(waypoints: Waypoints): Call =
     routes.BenefitsPaidEarlyController.onPageLoad(waypoints)
+
+  override protected def nextPageNormalMode(waypoints: Waypoints, answers: UserAnswers): Page = {
+    answers.get(BenefitsPaidEarlyPage) match {
```

Comment:
same comment above is valid for this too.
 `  override protected def nextPageNormalMode(waypoints: Waypoints, answers: UserAnswers): Page = {
    PaymentValueAndDatePage
  }`

Reply:
Resolved in next commit